cloudformation_templates/aws_nginx.json

```diff
@@ -288,9 +288,17 @@
         "AvailabilityZones" : {"Fn::GetAZs": ""},
         "LaunchConfigurationName" : {"Ref": "LaunchConfiguration"},
         "LoadBalancerNames": [{"Ref": "LoadBalancer"}],
+        "HealthCheckType": "ELB",
+        "HealthCheckGracePeriod": 300,
```

Comment:
5 minutes sounds like quite a long grace period. Does this affect how quickly it will apply the update?